### PR TITLE
#138 Add feature --include option 

### PR DESCRIPTION
**Pull request by Doldrums**
This is a fix for #138. I've added a basic implementation of feature --include option.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#152](https://github.com/yegor256/pdd/issues/152); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: .pdd [hunk 1]
```diff
@@ -18,5 +18,5 @@
 --exclude test/test_pdd.rb
 --exclude src/main/resources/images/**/*
 --rule min-words:20
---rule min-estimate:15
+--rule min-estimate:0
```
Comment (yegor256) [inline .pdd:21]: @Doldrums why? I suggest we keep it to 15

**Comment by yegor256** [comment]
@Doldrums one comment from me and I'm ready when you are

File: .pdd (hunk 1 above)
Comment (Doldrums) [reply .pdd:21]: oh sorry, that was used for testing. I will return now

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Doldrums try to run `bundle exec rake` on your local machine and make sure the build is clean. Now it's not, that's why can't merge (see log above).

**Comment by Doldrums** [comment]
@yegor256 better now?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Doldrums thanks!

**Pull request merged**